corehq/pillows/case.py

```diff
@@ -28,7 +28,14 @@
 pillow_logging.setLevel(logging.INFO)
 
 
+class InvalidChangeException(Exception):
+    pass
+
+
 def transform_case_for_elasticsearch(doc_dict):
+    if doc_dict.get('doc_type') == 'MobileAuthKeyRecord':
```

Comment:
any reason not to do an acceptlist instead of banning a specific type?

Will this cause the change to be retried via the retry queue?

Reply:
Yes
> This would raise a PillowError, with extra debug info and skip on these bad docs

Hoping to use that debug info to get some leads on root cause

Reply:
Agree with simon that this should be a whitelist of doc types